components/donation/Donation.component.tsx

```diff
@@ -47,16 +50,21 @@ type ParentProps = {
   show: (visible: boolean, message: string) => void;
 };
 
-export default function DonationView({ nodeInfo, show }: ParentProps) {
-  let addr: NodeAddress =
-    nodeInfo!.address.length > 0 ? nodeInfo!.address[0] : nodeInfo!.binding[0];
-  const [selectAddr, setSelectAddr] = useState(addr);
-  let mapAddress = new Map();
-  for (let i = 0; i < nodeInfo!.address.length; i++) {
-    let addr = nodeInfo!.address[i];
-    mapAddress.set(addr.type, addr);
-  }
-  if (nodeInfo === null) return <>NodeInfo null</>;
+export default function DonationView({ nodeInfo, show }: ParentProps,) {
+
+  // let addr: NodeAddress =
+  //   nodeInfo!.address.length > 0 ? nodeInfo!.address[0] : nodeInfo!.binding[0];
+  // const [selectAddr, setSelectAddr] = useState(addr);
+  // let mapAddress = new Map();
+  // for (let i = 0; i < nodeInfo!.address.length; i++) {
+  //   let addr = nodeInfo!.address[i];
+  //   mapAddress.set(addr.type, addr);
+  // }
+
+  const resp = useSWR("/api/offer", fetcher);
```

Comment:
the offer command has input, so we should be able to put this input too, we can use the core lightning.

So in this case I was thinking that maybe is better show the list of the invoice in a table and show the invoice in the QR code when click on a table row?

What do you think?